hack/containerd.toml

```diff
@@ -1,77 +1 @@
-root = "/var/lib/containerd"
-state = "/run/containerd"
-oom_score = 0
 subreaper = true
```

Comment:
This is the only option we are interested in. To avoid having to keep track with the full config between any two versions I think we should only specify the things we are interested in. This PR brings in Containerd 1.3.0.

Reply:
btw, in theory we don't need it anymore as we have reaper in machined

Reply:
oh ... I wonder if we should just use our reaper? I'm good with that if you are.

Reply:
we should give it a try :)